lib/chewy/config.rb

```diff
@@ -2,6 +2,15 @@ module Chewy
   class Config
     include Singleton
 
+    GLOBAL_OPTIONS = %i[
+      indices_path
+      prefix
+      index
+      wait_for_status
```

Comment:
I suppose this option should be client-specific, but let's keep this option global for now for simplicity